### PR TITLE
Take in a level id and return the level object endpoint

### PR DESCRIPTION
**Pull request by alyssacheong**

### CODE REVIEW

**Review by bengodw — COMMENTED** [review]
works perfect - I forgot to say though that this endpoint will also need to return the hotspots for the level, so you will need to create a hotspot interface (you can just copy this from the db), add an array of hotspots to the level interface and make it so that this endpoint also adds the hotspots to the level. This isn't a massive priority (like game still 'works' without hotspots) but I think Chris is implementing them on the frontend so would be good. Completely my fault for forgetting to raise this earlier.

File: backend/src/interfaces.ts [hunk 1]
```diff
@@ -18,6 +18,16 @@ export interface Level {
   latitude: number;
   longitude: number;
   zPosition: number | undefined;
+  hotspots: Hotspot[];
+}
+
+export interface Hotspot {
+  id?: string;
```
Comment (lachlanshoesmith) [inline backend/src/interfaces.ts]: is this ever undefined?

File: backend/src/interfaces.ts [hunk 2]
```diff
@@ -18,6 +18,16 @@ export interface Level {
...
+  photoLink: string;
+  locationName: string | undefined;
```
Comment (lachlanshoesmith) [inline backend/src/interfaces.ts]: every hotspot shoulllld have a name associated with it

**Review by lachlanshoesmith — CHANGES_REQUESTED** [review]
i think yellowbutton-related files can be deleted atp but i might be wrong

**Review by bengodw — APPROVED** [review]

**Pull request merged**